src/store/RecipesStore/RecipesStore.ts

```diff
@@ -59,14 +59,16 @@ export default class RecipesStore implements ILocalStore {
   async getRecipes(
     search: string | string[] | any | undefined,
     page: number,
-    offset: number
+    offset: number,
+    type?: string | string[] | any | undefined,
```

Comment:
я не уверен, но, мне кажется, необязательно здесь помечать аргумент необязательным (`type?: ...`), так как он делает лишь прибавление к имеющемуся перечислению типов `undefined`, хотя могу и ошибаться. но по крайней мере наоборот точно работает и можно сделать так без ущерба:
```suggestion
    type?: string | string[] | any,
```